dll/win32/shell32/iconcache.cpp

```diff
@@ -39,6 +27,9 @@ static HDPA        sic_hdpa = 0;
 
 static HIMAGELIST ShellSmallIconList;
 static HIMAGELIST ShellBigIconList;
+INT ShellSmallIconSize = 0;
+INT ShellBigIconSize = 0;
```

Comment:
s/Big/Large/
(which is the usual nomenclature for icon sizes)

Reply:
Fixed. Thanks.